setup.py

```diff
@@ -1,10 +1,9 @@
 #!/usr/bin/env python
 
-import vulcan
 from setuptools import setup
 
 setup(name='saltstack-vulcan',
-    version=vulcan.__version__,
+    version='0.1.2',
```

Comment:
Yeah, I didn't think of removing the imports from __init__ - this makes more sense. I'll re-refactor to just have the logging additions. Thanks!